<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-V6QH7CG" failures="1" tests="1" name="test1" time="18.854" errors="0" timestamp="2021-10-05T21:39:58 IDT">
  <testcase classname="Tests.ChatOnLine" name="ChatOnLineTest" time="10.965">
    <failure type="java.lang.AssertionError" message="expected [2] but found [1]">
      <![CDATA[java.lang.AssertionError: expected [2] but found [1]
at Japanika.Functions.IdentifyChat(Functions.java:21)
at Tests.ChatOnLine.ChatOnLineTest(ChatOnLine.java:45)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- ChatOnLineTest -->
</testsuite> <!-- test1 -->
